src/mpi/request/mpir_request.c

```diff
@@ -168,6 +168,15 @@ int MPIR_Request_completion_processing(MPIR_Request * request_ptr, MPI_Status *
                 break;
             }
 
+        case MPIR_REQUEST_KIND__PART_SEND:
+        case MPIR_REQUEST_KIND__PART_RECV:
+            {
+                MPID_Part_request_complete_hook(request_ptr);
```

Comment:
I am not sure about the goal of this hook either.

Reply:
It inactivates the req. But if we are going to move the `active` flag to `MPIR`, we can delete the hook. Will do.